types/node/index.d.ts

```diff
@@ -756,20 +756,23 @@ declare module "http" {
         write(str: string, encoding?: string, cb?: Function): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
 
-        writeContinue(): void;
-        writeHead(statusCode: number, reasonPhrase?: string, headers?: any): void;
-        writeHead(statusCode: number, headers?: any): void;
-        statusCode: number;
-        statusMessage: string;
+        addTrailers(headers: any): void;
+        finished: boolean;
+        getHeader(name: string): string;
+        getHeaderNames(): string[]
+        getHeaders(): any
```

Comment:
The reason I have used the any type for getHeaders is that getHeaders returns an object that prototypically inherits from null. Currently TypeScript doesn't allow to extend from type null. If you have any suggestions for a better type then please let me know.